src/backend/web/local/tests/bootstrap_test.py

```diff
@@ -249,12 +275,19 @@ def test_bootstrap_event(ndb_context, requests_mock: RequestsMocker) -> None:
         year=2020,
         team_list=[],
     )
+    matchstats = EventMatchstats(
+        oprs=make_team_stat_map(),
+        dprs=make_team_stat_map(),
+        ccwms=make_team_stat_map(),
+        coprs={},
```

Comment:
Do we want a test that includes `coprs` as well to make sure that round-trips properly?